unitTests/ui/CategoryFacet/CategoryFacetTestUtils.ts

```diff
@@ -0,0 +1,65 @@
+import { ICategoryFacetValue } from '../../../src/ui/CategoryFacet/CategoryFacetValues/CategoryFacetValue';
```

Comment:
Any reason why you did not put all those test utility function in the `Fake` module ?

Reply:
Seems more appropriate in something only for facets instead of a generic fake module, same for DynamicFacet...